include/UIChar.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef UICHAR_HPP
+# define UICHAR_HPP
+
+# include <string>
+
+class UIChar {
+
+	public:
+		UIChar(std::string const &);
+		UIChar(UIChar const &) = delete;
+		UIChar(UIChar &&) = delete;
+		UIChar &operator=(UIChar const &) = delete;
+		UIChar &operator=(UIChar &&) = delete;
+		virtual ~UIChar(void) = default;
+		
+		std::string getName(void) const;
+		void setScore(int);
```

Comment:
Avoid setters, use non-const getters that return references instead.
Also, if a variable has getters and setters that don't do anything more, it might be a good idea to make it public instead.